packages/demand-bloc-trip-create/package.json

```diff
@@ -23,7 +23,6 @@
   },
   "dependencies": {
     "@karhoo/demand-api": "^14.0.0",
-    "camelcase-keys": "^6.2.2",
```

Comment:
Does removal of this library relates to the fix directly? If no, are there some strong reasons for replacing it with the new utils functions?